src/main/java/uk/ac/ebi/eva/contigalias/controller/AdminController.java

```diff
@@ -62,16 +71,35 @@ public AdminController(AssemblyService service) {
-            "into local database.")
+            "into local database.",
+            notes = "Given an assembly's accession this endpoint will fetch and add the assembly that matches that " +
+                    "accession into the local database. The accession can be either a genbank or refseq accession and" +
+                    " the endpoint will automatically fetch the correct assembly from remote server. It will first " +
+                    "search for the target assembly in the local database as trying to insert an assembly which " +
+                    "already exists in the database is prohibited. If such an assembly is not found locally then it " +
+                    "will look for it at a remote source (NCBI by default). If desired assembly is found at remote " +
+                    "source, it will fetch and add it to the local database. This endpoint does not return any data.")
     @PutMapping(value = "v1/assemblies/{accession}")
-    public void fetchAndInsertAssemblyByAccession(@PathVariable String accession) throws IOException {
+    public void fetchAndInsertAssemblyByAccession(
+            @PathVariable @ApiParam(value = "Genbank or Refseq assembly accession. Eg: GCA_000001405.10") String accession) throws IOException {
         service.fetchAndInsertAssembly(accession);
     }
 
     @ApiOperation(value = "Fetch assemblies from remote server using their Genbank or Refseq accessions and insert " +
-            "into local database.")
+            "into local database.",
+            notes = "Given a list of assembly accessions, for every accession in the list this endpoint will fetch " +
+                    "and add the assembly that matches that accession into the local database. The accession can be " +
+                    "either a genbank or refseq accession and the endpoint will automatically fetch the correct " +
+                    "assembly from remote server. It will first search for the target assembly in the local database " +
+                    "as trying to insert an assembly which already exists in the database is prohibited. If such an " +
+                    "assembly is not found locally then it will look for it at a remote source (NCBI by default). If " +
+                    "desired assembly is found at remote source, it will fetch and add it to the local database. This" +
+                    " endpoint does not return any data and processes elements in the given list in an asynchronous " +
+                    "parallel manner.")
     @PutMapping(value = "v1/assemblies")
-    public void fetchAndInsertAssemblyByAccession(@RequestBody Optional<List<String>> accessions) {
+    public void fetchAndInsertAssemblyByAccession(@RequestBody @ApiParam(value =
+            "A JSON array of Genbank or Refseq assembly accessions. Eg: [\"GCA_000001405.10\",\"GCA_000001405.11\"," +
+                    "\"GCA_000001405.12\"]") Optional<List<String>> accessions) {
```

Comment:
now that I see this, I think it will be confusing to ask for a Optional<List<>> in the body, just ask for a List<>